ui/pages/routes/routes.component.js

```diff
@@ -117,6 +117,8 @@ export default class Routes extends Component {
         pageChanged(locationObj.pathname);
       }
     });
+
+    document.documentElement.setAttribute('data-theme', 'dark');
```

Comment:
Note: we need to remove this hardcoded theme code.

Reply:
Once https://github.com/MetaMask/metamask-extension/pull/13097 is merged, we can slip themes from `Settings > Experimental`

Reply:
rebased to include #13097